alpine-nix-rails-nginx/etc/sv/nginx

```diff
@@ -3,4 +3,6 @@
 set -o errexit -o pipefail -o noglob -o noclobber -o nounset
 IFS=$'\n\t'
 
-exec nginx -c /etc/nginx/nginx.conf -g 'daemon off;'
+export PATH="/nix/var/nix/profiles/default/bin:/nix/var/nix/profiles/default/sbin:$PATH"
```

Comment:
Since we are running the applications using `chpst` what I am doing is making sure that the `$PATH` is set to the absolute path rather than something user specific. This script is being run as root, the path will include `/root/.nix-profile/bin` and /root/.nix-profile/sbin`, but when the command is run as `rails` or `nginx` those users won't be able to access anything under `/root/`. By using the absolute path I can ensure any valid user can access the binaries.